src/DotNetCore.CAP/Persistence/IDbConnectionExtensions.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.ComponentModel;
+using System.Data;
+
+namespace DotNetCore.CAP.Persistence
+{
+    public static class IDbConnectionExtensions
```

Comment:
The extension method is added here and marked as public. When the user refers to this library, it may cause pollution of the extension method. I suggest copying this file to put it in the storage implementation project and changing it to internal

Reply:
I have considered this issue. But if put in the implementation, there will be a copy of the same code for each project.

Reply:
ok, I will modify it